### PR TITLE
rust activation 1.80 on main

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bollwyvl** [comment]
While waiting to build, a thought: would it be possible to provide (but then not use) some `source` so that this gets bot PRs? Also, presumably the linter is overzealous in this case until the hard deprecation of `MACOSX_DEPLOYMENT_TARGET`?

**Comment by bollwyvl** [comment]
@conda-forge/rust-activation ready for review. Wasn't sure what to do about the build number.

**Comment by bollwyvl** [comment]
On https://github.com/conda-forge/uv-feedstock/pull/67 I'm trying `rust_dev` and it appears to be working, but I guess it would be good to see this get in at some point...

**Comment by mbrobbel** [comment]
> @conda-forge/rust-activation ready for review. Wasn't sure what to do about the build number.

ping @isuruf @xhochy

**Comment by isuruf** [comment]
Can you bump the build number?

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

File: README.md [hunk 1]
```diff
@@ -323,14 +323,14 @@ Current release info
 Installing rust-activation
 ==========================
 
-Installing `rust-activation` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
+Installing `rust-activation` from the `conda-forge/label/rust_dev` channel can be achieved by adding `conda-forge/label/rust_dev` to your channels with:
```
Comment (bollwyvl) [inline README.md:326]: eee, this looks weird. but then, not so many folk knowingly _install_ this package...

Comment (isuruf) [reply README.md:326]: Yeah, that's okay

**Pull request merged**